package.json

```diff
@@ -7,14 +7,16 @@
   },
   "scripts": {
     "start": "npm run web",
+    "analyze2": "source-map-explorer 'build/static/js/*.js'",
```

Comment:
Should't we use `analize:something` here as in start? I would suggest `analyze:source-map` and `analyze:bundle`

Reply:
done